yaksh/templates/yaksh/user_data.html

```diff
@@ -1,271 +1,300 @@
-              {% endfor %}
-              </div>
-
-            {% else %}
-                <h5> <u>Test cases: </u></h5>
-                    {% for testcase in question.get_test_cases %}
-                        <br/><strong>{{ forloop.counter }}. {{ testcase.test_case }}</strong>
-                    {% endfor %}
-            {%endif%}
-        </div>
-    </div>
-    {% if question.type != "code" %}
-        {% if "Correct answer" in answers.0.error_list %}
-            <div class="panel panel-success">
-            <div class="panel-heading"><strong>Correct</strong></div>
-        {% else %}
-            <div class="panel panel-danger">
-            <div class="panel-heading"><strong> Incorrect</strong></div>
-        {% endif %}
-            <div class="panel-body">
-                <h5><u>Student answer:</u></h5>
-
-        {% if question.type == "mcc"%}
-        <div class="well well-sm">
-        {% for testcases in question.get_test_cases %}
-        {%if testcases.id|stringformat:"i" in answers.0.answer|safe %}
-         <li>{{ testcases.options.strip|safe }}</li>
-        {% endif %}
-        {% endfor %}
-        </div>
-
-        {% elif question.type == "mcq"%}
-        <div class="well well-sm">
-        {% for testcases in question.get_test_cases %}
-        {%if testcases.id|stringformat:"i" == answers.0.answer|safe %}
-         <li>{{ testcases.options.strip|safe }}</li>
-        {% endif %}
-        {% endfor %}
-        </div>
-
-        {% elif question.type == "arrange"%}
-        <div class="well well-sm">
-          {% get_answer_for_arrange_options answers.0.answer question as tc_list %}
-        {% for testcases in tc_list %}
-         <li>{{ testcases.options.strip|safe }}</li>
-        {% endfor %}
-        </div>
-
-        {%else%}
-                 <div class="well well-sm">
-                 {{ answers.0.answer|safe }}
-                 </div>
-        {% endif %}
+   <div class="card col-md-4">
+      <p class="card-body"> 
+         Name: {{ data.user.get_full_name.title }} <br/>
+         Username: {{ data.user.username }} <br/>
+         {% if data.profile %}
+         Roll number: {{ data.profile.roll_number }} <br/>
+         Position: {{ data.profile.position }} <br/>
+         Department: {{ data.profile.department }} <br/>
+         Institute: {{ data.profile.institute }} <br/>
+         {% endif %}
+         Email: {{ data.user.email }} <br/>
+         Date joined: {{ data.user.date_joined }} <br/>
+         Last login: {{ data.user.last_login }}
+      </p>
+   </div>
+<!-- </div> -->
+   {% if data.papers %}
+   <p class="text-center"><a href="{{URL_ROOT}}/exam/manage/gradeuser/{{data.papers.0.question_paper.quiz.id}}/{{ data.user.id }}/{{course_id}}/"  class="btn btn-info ">
+      Grade/correct paper</a>
+   </p>
+   {% for paper in data.papers %}
+   {% if forloop.counter == 2 and data.questionpaperid %}
+   <hr>
+   <U>
```

Comment:
Only if there is another commit to follow then this ```u tag``` can be lowercase for consistency purpose

Reply:
Done